ctl/version/version.go

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright The Kmesh Authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at:
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package version
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"kmesh.net/kmesh/ctl/utils"
+	"kmesh.net/kmesh/pkg/logger"
+	"kmesh.net/kmesh/pkg/version"
+)
+
+var log = logger.NewLoggerScope("kmeshctl/version")
+
+func NewCmd() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:   "version",
+		Short: "Prints out build version info",
+		Example: `# Show version of kmeshctl
+kmeshctl version
+
+# Show version info of a specific Kmesh daemon
+kmesh version <kmesh-daemon-pod>`,
+		Run: func(cmd *cobra.Command, args []string) {
+			_ = RunVersion(cmd, args)
+		},
+	}
+	return cmd
+}
+
+// RunVersion provides the version info of kmeshctl or specific Kmesh daemon.
+func RunVersion(cmd *cobra.Command, args []string) error {
+	if len(args) == 0 {
+		v := version.Get()
+		fmt.Printf("%s\n", v.GitVersion)
```

Comment:
since the CLI tool is built with cobra, is there a reason we shouldn't use the cobra `cmd.Printf()`, `cmd.PrintErrf()`, etc methods?

Reply:
good catch! I'll look into it :)